assets/app/view/game/round/stock.rb

```diff
@@ -406,18 +405,11 @@ def render_select_par_slot
           h(:div, children)
         end
 
-        def render_take_loan
-          take_loan = lambda do
-            process_action(Engine::Action::TakeLoan.new(@current_entity, loan: nil))
-          end
-          [h(:button, { on: { click: take_loan } }, "Take Loan (#{@game.format_currency(@game.loan_amount)})")]
-        end
-
-        def render_payoff_loan
-          payoff_loan = lambda do
-            process_action(Engine::Action::PayoffLoan.new(@current_entity, loan: nil))
+        def render_game_buttons
+          @step.game_buttons(@current_entity).map do |game_button|
```

Comment:
render return is expected to be an html element, not an array. The array needs to be wrapped in a div.

Reply:
Done.  I think you mentioned this before and I didn't change it.  Not sure why.